### ISSUE TITLE
multiembed support

### ISSUE THREAD

**Issue by kolekan** [Gujal00/ResolveURL#675]
Please can you add support for links like `https://multiembed.mov/directstream.php?video_id=tt11145118` ?
Thanks in advance

**Comment by Gujal00** [comment]
That is an embedder and not an hoster, you need to scrape further to find the hoster url and pass it to resolveurl.
From looking at that particular embed above, the m3u8 manifest they present uses non standard extensions for the segments due to which kodi wont play
```
#EXTM3U
#EXT-X-TARGETDURATION:12
#EXT-X-ALLOW-CACHE:YES
#EXT-X-PLAYLIST-TYPE:VOD
#EXT-X-VERSION:3
#EXT-X-MEDIA-SEQUENCE:1
#EXTINF:3.837,
seg-1-v1-a1.jpg
#EXTINF:10.427,
seg-2-v1-a1.html
#EXTINF:2.711,
seg-5-v1-a1.js
#EXTINF:1.460,
seg-6-v1-a1.css
#EXTINF:11.803,
seg-7-v1-a1.txt
#EXTINF:3.420,
seg-11-v1-a1.png
#EXTINF:5.589,
seg-12-v1-a1.webp
#EXTINF:6.798,
seg-14-v1-a1.ico
```

**Comment by MrDini123** [comment]
> due to which kodi wont play

My Kodi plays it just fine. However there is an issue with the regular Kodi player for HLS streams. It lags and often hangs to load the next chunks. I have that happen across all versions of Kodi on different platforms and only with HLS. So our solution was to switch to ISA (inputstream.adaptive) for HLS streams. On 20.2 ISA works just fine with custom extensions in the HLS streams, but you are right, it's broken on 19.5 and below. Only with ISA though.

I experience the same issue with waaw now as they use `.mp666` and not `.ts` in their HLS streams. I use ISA and 20.2 users can play it just fine. 19.5 users couldn't however. So I wrote a local webserver that serves a modified manifest with `.ts` endings and the chunk serving endpoint just throws a 302 redirect to the actual chunk URL at the hoster. Works fine.

So I believe it would be nice to have that provider in ResolveURL as 20.2 users can play those streams just fine and users who don't mind using the built-in player can play it below 20.2 as well. Or they can always go the hacky way that I explained above. Although I won't use this provider personally...
